include/avif/internal.h

```diff
@@ -15,26 +15,44 @@ extern "C" {
 #define AVIF_MIN(a, b) (((a) < (b)) ? (a) : (b))
 #define AVIF_MAX(a, b) (((a) > (b)) ? (a) : (b))
 
+// Used for debugging. Add ! to catch the earliest failure during encoding or decoding.
```

Comment:
Optional: I thought we were supposed to add a `#define AVIF_BREAK_ON_ERROR` line to enable this. If we enable it by modifying line 19, it seems clearer to rewrite line 19 as `#if 0` and ask users to change it to `#if 1` to enable it.